### ISSUE TITLE
Problem with adding excessive parentheses and deleting essential parentheses

### ISSUE THREAD

**Issue by hstarorg** [ThisIsManta/vscode-stylus-supremacy#14]
Stylus Code: 
```stylus
$base = 1 / 24 * 100;

for i in 1 .. 24 {
  .sa-col-{i} {
    width: ($base * i)%;
  }
}
```
After Format: 
```stylus
$base = (1 / 24) * 100;

for i in 1 .. 24 {
  .sa-col-{i} {
    width: $base * i %;
  }
}
```
After Second Format:
```stylus
$base = ((1 / 24)) * 100;

for i in 1 .. 24 {
  .sa-col-{i} {
    width: $base * i %;
  }
}
```

**Comment by ThisIsManta** [comment]
I'll take care of that, but I cannot give you a promising date.

**Comment by hstarorg** [comment]
@ThisIsManta Very thanks. Maybe it's hard for achieve.

**Comment by ThisIsManta** [comment]
Fixed in version [2.3.0](https://github.com/ThisIsManta/stylus-supremacy/releases/tag/v2.3.0) on Node.js and VSCode.

Formatting parentheses must be better now. However, I made tremendous changes in the code, hence, this may introduce another bug. Please report if you found any.

Cheers!

**Comment by hstarorg** [comment]
@ThisIsManta Perfect, so fast.
Cheers~
